Libs/MRML/DisplayableManager/vtkThreeDViewInteractorStyle.cxx

```diff
@@ -792,6 +783,56 @@ void vtkThreeDViewInteractorStyle::SetModelDisplayableManager(
 }
 
 //----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse)
```

Comment:
Sounds good. I'll look into that. However, it looks like `AxisLabel` can be modified to customize the labels with `SetAxisLabel`. Should we pass an integer which defined the index of the axis instead? Or just go through all the values of `AxisLabel` and check if it exists, then infer how to rotate based on the axis id?

Also, @vovythevov, do you remember why you implemented this in the interactor style as well as the qMRMLThreeDView? Thank you.